encoding.bs

```diff
@@ -15,6 +15,19 @@ Translate IDs: dictdef-textdecoderoptions textdecoderoptions,dictdef-textdecodeo
 spec:infra; type:dfn;
     text:code point
     text:ascii case-insensitive
+spec:streams;
+    type:interface; text:ReadableStream
+    type:dfn; text:chunk
```

Comment:
We should export these in streams so that this is not necessary, for the dfns at least. For the interface, that's a separate issue: https://github.com/whatwg/fetch/issues/780 so we can leave it here for now.